terraso_backend/apps/graphql/schema/groups.py

```diff
@@ -95,3 +95,15 @@ class GroupDeleteMutation(BaseDeleteMutation):
 
     class Input:
         id = graphene.ID()
+
+    @classmethod
+    def mutate_and_get_payload(cls, root, info, **kwargs):
+        user = info.context.user
+
+        ff_check_permission_on = settings.FEATURE_FLAGS["CHECK_PERMISSIONS"]
```

Comment:
Can we remove the feature flag and assume a concurrent deploy of the frontend work?

Reply:
I would rather keep the feature flag until we make the proper changes on web client.

Reply:
Thought: I like feature flags, I think the important process here will be how to maintain it later, maybe at some point it is not needed and we should remove the flag. For this particular situation since a coordination is required between client and api I think it makes sense to keep it